AnnoyancesFilter/sections/popups.txt

```diff
@@ -91,6 +91,7 @@ sodapdf.com##.be-tooltip
 povarenok.ru##.dr
 versii.if.ua###lightb_fb_page_like_baner
 versii.if.ua##.overlay_fon
+dugtor.ru##div[style*="position:fixed"][style*="width:488px"][style*="z-index:1000011"]
```

Comment:
There is no need to block element because it's better to block push-notification `||heroesdom.com^$third-party` in `Annoyances -> push-notifications.txt`